### DISCUSSION TITLE
Sending logs from Dart (Flutter) to Seq?

### DISCUSSION THREAD

**Discussion by yooakim** [datalust/seq-tickets#1868] [General]
I'm a long time user of Seq (often with Serilog) but now I am helping a client that is doing greenfield development.

The new system is using a REST API built using .NET Core and a client application written in Dart/Flutter.

We have Seq for the back end but would really like to get also the Dart code to send logs to Seq.

I am sure this could be done... but I am wondering if someone has good pointers or care to share experience doing this?

**Comment by liammclennan** [comment]
Hi @yooakim 

There does not seem to be an Open Telemetry client for flutter, and there is not a Seq one, so the only option left (that I know of) is to send events to the [Seq HTTP API](https://docs.datalust.co/docs/server-http-api#apievents). You can have a look at the implementation in [seq-logging](https://github.com/datalust/seq-logging/blob/dev/seq_logger.js#L245) to get some ideas.

**Comment by Mike-E-angelo** [comment]
As this thread returned at the top of the list when searching for "Datalust Seq Flutter" I thought I'd update to say that this appears to be supported in some capacity by the Dart ecosystem now:

https://pub.dev/packages/seq_logger

**Reply by nblumhardt** [reply]
Very cool! Thanks @Mike-E-angelo 👍
